gorm-tools-rest/src/main/groovy/gorm/tools/rest/controller/RestRepositoryApi.groovy

```diff
@@ -172,6 +179,24 @@ trait RestRepositoryApi<D> implements RestApiController {
         respond([data: data])
     }
 
+    /** Used for bulk create calls when Job object is returned */
+    // XXX https://github.com/9ci/domain9/issues/331 Fix it so it works with Job that is created
+    @Action
+    def bulk() {
```

Comment:
keep names consistent

Reply:
I think I like your idea of having one endpoint called bulk, single bulk endpoint will work however, as its stand it only supports create, so you would still need to have a bulkCreate and a bulkUpdate in the controller and we would map /bulk POST onto one and bulk PUT onto the other in urlMappings.